mmv1/third_party/terraform/resources/resource_storage_bucket.go.erb

```diff
@@ -343,7 +343,14 @@ func resourceStorageBucket() *schema.Resource {
 				Computed:    true,
 				Description: `Enables uniform bucket-level access on a bucket.`,
 			},
+<% unless version == "ga" -%>
+			"public_access_prevention": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: `Prevents public access to a bucket.`,
+			},
 		},
+<% end -%>
```

Comment:
Right... but this block only adds the additional schema field if we're in beta mode... isn't that what we mean by beta being a superset of ga? The proposed change here looks to me like it would add the field to the schema for both providers.

Reply:
Thats correct. I'm confused your original proposed change was against terraform-provider-google. Not the beta variant. Are you saying this field does not exist on the GA version and should not be added there?

Reply:
That's my understanding.... it's flagged in Google's doc as a preview feature:

https://cloud.google.com/storage/docs/public-access-prevention

>This feature is covered by the Pre-GA Offerings Terms of the Google Cloud Terms of Service. Pre-GA features might have limited support, and changes to pre-GA features might not be compatible with other pre-GA versions. For more information, see the launch stage descriptions.

Reply:
ah I see, then this and your previous test file should be accurate. Could you revert the test files to your previous iteration ?

Reply:
Sure thing, coming up...